foundations/javascript_basics/fundamentals-5.md

```diff
@@ -1,5 +1,7 @@
 ### Introduction
 
+Congratulations on making it to the final lesson in foundations! By this point, you have learned many of the fundamentals of JavaScript. In this lesson, you will learn about Objects - "a collection of key-value pairs." as well as some more powerful and commonly used array methods. Before you know it, you'll have a better understanding of how powerful objects and arrays are and how both can be an indispensable part of your JavaScript tool kit!
```

Comment:
🎉 Awesome job on this! Had a few tweaks below, let me know what you think!

```suggestion
Congratulations on making it to one of the last lessons in Foundations! By this point, you have learned many of the fundamentals of JavaScript. In this lesson, you will learn about Objects - a collection of key-value pairs - as well as some more powerful and commonly used array methods. Before you know it, you'll have a better understanding of how powerful objects and arrays are and how both can be an indispensable part of your JavaScript tool kit!
```

Mainly changing "final lesson" to "one of the final lessons", just because of technicalities

Reply:
Thank you! Great point on the technicalities. I appreciate your feedback. Looks superb now! :)

Reply:
Has the suggestion been applied by me clicking the commit suggestion drop down, or do I need to add the changes via my local branch and then merge?